web_responsive/static/lib/js/drawer.3.2.0.js

```diff
@@ -136,7 +136,11 @@
       return $this
         .removeClass(__.settings.class.open)
         .addClass(__.settings.class.close)
-        .css({ 'overflow': 'auto' })
+        // TODO: remove this local patch once PR below is approved:
```

Comment:
Given [@Yajo's comment on 511](https://github.com/OCA/web/pull/511#issuecomment-269595936), This line should probably be just removed, and the following block added to `web_responsive/static/lib/css/drawer.3.2.0.css`:

    .drawer-open {
        overlay: hidden;
    }

After doing the same in a PR for drawer, of course (which means, I should probably [update my own PR](https://github.com/blivesta/drawer/pull/36)...)